README.md

```diff
@@ -115,7 +115,7 @@ A list of relevant skills: List the programming languages, frameworks, and tools
 
 > Add a link to your deployed project.
```

Comment:
[OPTIONAL] I think you can do away with this line. it is making your readme still look like a template😉